Mastodon/Scene/Discovery/ForYou/ProfileCardView.swift

```diff
@@ -316,9 +316,9 @@ extension ProfileCardView {
                 buttonState = .none
             } else if relationship.following {
                 buttonState = .unfollow
-            } else if relationship.blocking || (relationship.domainBlocking ?? false) {
+            } else if relationship.blocking || relationship.domainBlocking {
                 buttonState = .blocked
-            } else if relationship.requested ?? false {
+            } else if relationship.requested {
```

Comment:
😍 😍

Reply:
I checked with the API and decided to make several properties non-optional